exercise_30.js

```diff
@@ -0,0 +1,15 @@
+/*
+Implement the isLeapYear  function it should accept one argument:
+
+a number
+
+the function should return true  if the number represents a leap year, and false otherwise
+ */
+
+function isLeapYear(number) {
+    return number === 366;
+}
+
+
+console.log(isLeapYear(365));
+console.log(isLeapYear(366));
```

Comment:
w tym zadaniu chodzi o to żeby podać rok i dostać informacje czy jest on robiem przestępnym, w twoim rozwiązaniu tylko rok 366 jest rokiem przestępnym co NIE jest prawdą